code/modules/clothing/suits/armor_suits.dm

```diff
@@ -753,45 +753,45 @@
 //Non-hardsuit ERT armor.
 /obj/item/clothing/suit/armor/vest/ert
 	name = "emergency response team armor"
-	desc = "A set of armor worn by members of the Nanotrasen Emergency Response Team."
+	desc = "A mid-quality protective vest produced by Citadel Armories. Additional polymer paneling over the chest and shoulders offers moderately improved energy protection compared to standard kevlar vests."
```

Comment:
Since this is ERT gear, and has better stats than protective kevlar vests (correct me if I'm wrong), wouldn't it be a bit disingenuous to call this "mid quality" ? I feel like it being called "good" would work, as ERTs are elite, even if they're of the Amber variety.

Reply:
The Amber vest is identical statwise to a standard sec vest save increased energy protection, hence the mention of additional paneling.

Reply:
Gotcha, then how about something like "A well-kept protective vest [...]" ? It would highlight the wearer's (supposed) experience, and make it appear less cheap.